src/create-comment.ts

```diff
@@ -0,0 +1,47 @@
+import { getOctokit } from '@actions/github';
+type CreateCommentParams = {
+  token: string;
+  owner: string;
+  repo: string;
+  prNumber: number;
+};
+
+export async function createComment(
+  params: CreateCommentParams,
+  body: string,
+  block: boolean
+) {
+  const { token, owner, repo, prNumber } = params;
+  const octokit = getOctokit(token);
+
+  const { data: allComments } = await octokit.rest.issues.listComments({
+    owner,
+    repo,
+    issue_number: prNumber
+  });
+
+  const comment = allComments.find(item => {
+    return (
+      item.user?.login === 'github-actions[bot]' &&
+      item.body?.startsWith('## Todo Commenter\n')
+    );
+  });
```

Comment:
- [ ] This needs to be fixed, for updating the existing comment
- [ ] We also need functionality for deleting comments if there is nothing to be done